invokeai/backend/assets/model_base_conf_files/controlnet_sd15/config.json

```diff
@@ -0,0 +1,42 @@
+{
+  "_class_name": "ControlNetModel",
+  "_diffusers_version": "0.16.0.dev0",
+  "_name_or_path": "/home/patrick/controlnet_v1_1/control_v11p_sd15_canny",
```

Comment:
I ran an sd1 controlnet as a single file and it didn't seem to break as a result of this, but it seems weird to have this hard coded filepath here

Reply:
The _name_or_path field is ignored during file conversion, but these fields will be removed.